src/components/BeachPage/BeachReviews/BeachReviews.js

```diff
@@ -20,27 +20,24 @@ async function getData(id) {
   }
   
 const BeachReviews = (props) => {
-    const router = useRouter();
-    const { beachid } = router.query
+    const { beachid } = props;
```

Comment:
because we're now using the slug format, i realized that I should use `router.query` as sparingly as possible, so that components can be self contained and all the router logic just lives in the root page file